includes/admin/settings/views/html-admin-page-shipping-zone-methods.php

```diff
@@ -97,8 +98,12 @@
 				</table>
 			</td>
 		</tr>
+	<?php do_action( 'woocommerce_shipping_zone_inside_bottom_methods_table' ); ?>
 	</tbody>
```

Comment:
This looks good. Thanks for your contribution.

One small change please, then this is good to merge: The indenting of the `woocommerce_shipping_zone_inside_bottom_methods_table` and `woocommerce_shipping_zone_inside_top_methods_table` actions is inconsistent. Please indent `woocommerce_shipping_zone_inside_bottom_methods_table` one more tab (so it lines up with the `</tr>` on the previous live).

Reply:
done :+1: